www/layouts/about/section.html

```diff
@@ -366,7 +367,7 @@ <h1 class="text-center-mobile pb-5">Beyond Course Materials</h1>
               self-guided and includes no live support, discussion forum, or
               certificates.
             </p>
-            <a class="red-btn" href="/search">OCW Search</a>
+            <a class="red-btn" href="/course_collections/open-learning-library/">Open Learning Library courses</a>
```

Comment:
```suggestion
            <a class="red-btn" href="/pages/open-learning-library/">Open Learning Library courses</a>
```

We changed this to a page instead of a course collection